### DISCUSSION TITLE
Some confusion about Pole figure

### DISCUSSION THREAD

**Discussion by louis96108** [mtex-toolbox/mtex#939] [Ask Anything] [answered]
Hi，everyone
   Recently, I used MTEX to calculate the Child variants in Titanium. Now，I copy the code and the command window here.
```
csBeta = crystalSymmetry('432',[3.3 3.3 3.3],'mineral','Ti (beta)');
csAlpha = crystalSymmetry('622',[3 3 4.7],'mineral','Ti (alpha)');
oriParent = orientation. byEuler(151*degree,34*degree,27*degree,csBeta);
beta2alpha = orientation.map(Miller(1,-1,0,csBeta),Miller(0,0,0,1,csAlpha),...
      Miller(1,1,1,csBeta),Miller(1,1,-2,0,csAlpha));
oriChild = oriParent * inv(beta2alpha)
hAlpha = Miller({0,0,0,1},{1,1,-2,0},csAlpha);
hBeta  = Miller({1,0,1},{1,1,1},csBeta);
plotPDF(oriParent,hBeta,'MarkerSize',20,'MarkerFaceColor','none','linewidth',4)
nextAxis(1)
hold on
plot(oriChild * hAlpha(1).symmetrise,'MarkerSize',12)
xlabel(char(hAlpha(1)),'color',ind2color(2))
nextAxis(2)
plot(oriChild * hAlpha(2).symmetrise,'MarkerSize',12)
xlabel(char(hAlpha(2)),'color',ind2color(2))
hold off
drawNow(gcm)
---------------------------------------------
oriChild = orientation (show methods, plot)
  crystal symmetry : Ti (alpha) (622, X||a*, Y||b, Z||c*)
 
  Bunge Euler angles in degree
     phi1     Phi    phi2    Inv.
  225.924 99.9507 37.9444       0
```
The Pole Figures are just like these
(https://user-images.githubusercontent.com/69557894/113873110-12961f80-97e7-11eb-8331-dd2ad4431845.png)

Then I plotted the {11-20} Pole figure in 'Texviewer' and its result is the same as 'OIM'. I paste the picture here.
(https://user-images.githubusercontent.com/69557894/113873762-aff15380-97e7-11eb-8e46-1068af62ee8d.png)

It could be observed that these two pole figures are not consistent. Maybe someone knows why. Looking forward to hearing from you.Best wishes to you!

Sincerely,
Louis

**Comment by kilir** [comment, marked as answer]
Hi,
I think you use a different definition for the hexagonal symmetry:
```
csAlpha = crystalSymmetry('622')
```
vs
```
csAlpha = crystalSymmetry('622','X||a')
```

See here: https://mtex-toolbox.github.io/CrystalReferenceSystem.html

Cheers,
Rüdiger

**Reply by louis96108** [reply]
Hi，Rüdiger
 There are indeed some problems with symmetry and I’ve solved the problem. Thanks for your reply.

Cheers,
Louis
